projects/1.21/assets/super-factory-manager/sfm/lang/zh_cn.json

```
@@ -0,0 +1,238 @@
+{
+  "block.sfm.cable": "存储线缆",
```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Inventory Cable**

```suggestion
  "block.sfm.cable": "工厂线缆",
```

---

其实`存储线缆`和`工厂线缆`感觉都不太行
不知道有没有其他好的方案

---

根据语义，这里是表示对 Inventory / 容器的访问（特性命名），鉴于该操作较为抽象，用 `工厂线缆` 也不过不失，或可以作为对模组特质、物品在体系中的重要程度等要素的彰显？

---

那就工厂线缆吧，原翻译可能引起误解